Foodie/Order.rb

```diff
@@ -0,0 +1,53 @@
+class Order
+  attr_accessor :items, :oid, :cust_nm, :status 
+  @@ocnt=1
+  def initialize
+    @oid=@@ocnt
```

Comment:
add spaces around operators. "=" is operator here. 

you already have `attr_accesso`r for `cust_nm` & `status` and if you do not want them to initialise with specific value, no need to add them to `initialize` block

also use better variable names, cust_nm & ocnt are not readable.